proto/xmtpv4/message_api/message_api.proto

```diff
@@ -159,9 +102,9 @@ service ReplicationApi {
   }
 
   // Publish envelope
-  rpc PublishEnvelopes(PublishEnvelopesRequest) returns (PublishEnvelopesResponse) {
+  rpc PublishPaidEnvelopes(PublishPaidEnvelopesRequest) returns (PublishPaidEnvelopesResponse) {
```

Comment:
I didn't want to have two APIs named `PublishEnvelopes` that expect different request/response types. That feels like a footgun.

This is the best name I could come up with.

Reply:
I'm wondering if `PublishPayerEnvelopes` and `PublishClientEnvelopes` would be more consistent?